module_utils/network_lsr/nm_provider.py

```diff
@@ -21,3 +22,18 @@ def get_nm_ethtool_feature(name):
 
     feature = getattr(Util.NM(), name, None)
     return feature
+
+
+def get_nm_ethtool_setring(name):
```

Comment:
Maybe `get_nm_ethtool_set_ring`?

Reply:
Is `set` really part of option? It seems to be just `ring` (compare to feature).

Reply:
Changed into `get_nm_ethtool_ring`.